policy/policy.go

```diff
@@ -433,6 +433,7 @@ func (p *Policy) updateAllChecksums(ctx context.Context,
 			}
 
 			executionChecksum = executionChecksum.Add(ref.Mrn)
+			executionChecksum = executionChecksum.AddUint(uint64(ref.Action))
```

Comment:
Does the line right above this have any purpose if the result in `executionChecksum` is immediately overwritten?

Reply:
It's not overwritten, it get's additional data added:
https://github.com/mondoohq/cnquery/blob/323c116a72e37534ebeafc2ccfe9c6bc71f79e7b/checksums/checksum.go#L22

Another example for this in this file:
https://github.com/mondoohq/cnspec/blob/e730ccf05254e433192557aa73eb98effc123688/policy/policy.go#L366-L376